TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Persons/PersonDetail/EditInduction/CommonPageTests.cs

```diff
@@ -0,0 +1,184 @@
+using AngleSharp.Html.Dom;
+using TeachingRecordSystem.SupportUi.Pages.Persons.PersonDetail.EditInduction;
+
+namespace TeachingRecordSystem.SupportUi.Tests.PageTests.Persons.PersonDetail.EditInduction;
+
+public class CommonPageTests(HostFixture hostFixture) : TestBase(hostFixture)
+{
+    [Theory]
+    [InlineData("edit-induction/status", InductionStatus.Exempt, "edit-induction/exemption-reasons")]
+    [InlineData("edit-induction/status", InductionStatus.InProgress, "edit-induction/start-date")]
+    [InlineData("edit-induction/status", InductionStatus.Failed, "edit-induction/start-date")]
+    [InlineData("edit-induction/status", InductionStatus.FailedInWales, "edit-induction/start-date")]
+    [InlineData("edit-induction/status", InductionStatus.Passed, "edit-induction/start-date")]
+    [InlineData("edit-induction/status", InductionStatus.RequiredToComplete, "edit-induction/change-reason")]
+    [InlineData("edit-induction/exemption-reasons", InductionStatus.Exempt, "edit-induction/change-reason")]
+    [InlineData("edit-induction/start-date", InductionStatus.InProgress, "edit-induction/change-reason")]
+    [InlineData("edit-induction/start-date", InductionStatus.Failed, "edit-induction/date-completed")]
+    [InlineData("edit-induction/start-date", InductionStatus.FailedInWales, "edit-induction/date-completed")]
+    [InlineData("edit-induction/start-date", InductionStatus.Passed, "edit-induction/date-completed")]
+    [InlineData("edit-induction/date-completed", InductionStatus.Failed, "edit-induction/change-reason")]
+    [InlineData("edit-induction/date-completed", InductionStatus.FailedInWales, "edit-induction/change-reason")]
+    [InlineData("edit-induction/date-completed", InductionStatus.Passed, "edit-induction/change-reason")]
+    [InlineData("edit-induction/change-reason", InductionStatus.Exempt, "edit-induction/check-answers")]
+    [InlineData("edit-induction/change-reason", InductionStatus.InProgress, "edit-induction/check-answers")]
+    [InlineData("edit-induction/change-reason", InductionStatus.Failed, "edit-induction/check-answers")]
+    [InlineData("edit-induction/change-reason", InductionStatus.FailedInWales, "edit-induction/check-answers")]
+    [InlineData("edit-induction/change-reason", InductionStatus.Passed, "edit-induction/check-answers")]
+    [InlineData("edit-induction/change-reason", InductionStatus.RequiredToComplete, "edit-induction/check-answers")]
+    [InlineData("edit-induction/check-answers", InductionStatus.RequiredToComplete, "induction")]
+    public async Task Post_RedirectsToExpectedPage(string fromPage, InductionStatus inductionStatus, string expectedNextPageUrl)
+    {
+        // Arrange
+        var person = await TestData.CreatePersonAsync(
+            personBuilder => personBuilder
```

Comment:
We tend to use short parameter names here
```suggestion
            p => p
```